crates/modelardb_compression/src/models/swing.rs

```diff
@@ -84,9 +89,12 @@ impl Swing {
     /// - (2) When the second data point is received, two linear functions that
     /// intersect with the first data point are computed to designate the upper
     /// and lower bounds for the linear functions Swing can fit to the segment.
+    /// From the second data point, the recording mechanism that is used to
```

Comment:
Unclear what "the recording mechanism is updated" mean, do you mean minimum slope is updated? Maybe simplify doc comment.

Reply:
It is still unclear what "recording mechanism" means in this context, can the comment be rewritten so it is removed?